diff --git a/ld/ld.texinfo b/ld/ld.texinfo
index c89915f1aa..f0f2a013d7 100644
--- a/ld/ld.texinfo
+++ b/ld/ld.texinfo
@@ -2409,7 +2409,7 @@ systems may not understand them. If you specify
 @option{--enable-new-dtags}, the new dynamic tags will be created as needed
 and older dynamic tags will be omitted.
 If you specify @option{--disable-new-dtags}, no new dynamic tags will be
-created. By default, the new dynamic tags are not created. Note that
+created. By default, the new dynamic tags are created. Note that
 those options are only available for ELF systems.
 
 @kindex --hash-size=@var{number}
diff --git a/ld/ldmain.c b/ld/ldmain.c
index e02fd3f1ff..dccc76b3cc 100644
--- a/ld/ldmain.c
+++ b/ld/ldmain.c
@@ -271,6 +271,7 @@ main (int argc, char **argv)
 
   link_info.allow_undefined_version = TRUE;
   link_info.keep_memory = TRUE;
+  link_info.new_dtags = TRUE;
   link_info.combreloc = TRUE;
   link_info.strip_discarded = TRUE;
   link_info.emit_hash = DEFAULT_EMIT_SYSV_HASH;
